source/administration/command-line-tools.rst

```diff
@@ -655,3 +660,181 @@ platform version
     .. code-block:: none
 
       platform version
+
+Mattermost 3.5 and earlier
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typing ``platform -help`` brings up documentation for the CLI tool.
+
```

Comment:
Thanks Jeff. Are you proposing every command leads with `sudo ./platform`? Or could we just add something to the doc intro section to explain this?